src/data/ab-attrs/stat-multiplier-ab-attr.ts

```diff
@@ -58,7 +59,8 @@ export class StatMultiplierAbAttr extends AbAttr {
     args: any[],
   ): boolean {
     const move = args[0] as Move;
-    if (stat === this.stat && (!this.condition || this.condition(pokemon, null, move))) {
+    const target = args[1] as Pokemon;
+    if (stat === this.stat && (!this.condition || this.condition(pokemon, target, move))) {
```

Comment:
```suggestion
    const opponent = args[1] as Pokemon;
    if (stat === this.stat && (!this.condition || this.condition(pokemon, opponent, move))) {
```

to match where this is applied in `Pokemon#getEffectiveStat`